### DISCUSSION TITLE
Any way to create a collection for TV episodes with commentary?

### DISCUSSION THREAD

**Discussion by hexfield** [meisnate12/Plex-Meta-Manager#723] [Q&A] [answered]
Trying to make a collection with just episodes of TV shows that have commentary tracks. I'm doing this with movies already using a filter for "audio_track_title: Commentary" but it looks like filters aren't available for episode level collections? Any workaround for this? Thanks!

**Comment by meisnate12** [comment, marked as answer]
filters are being made available to episodes in the next release

**Reply by hexfield** [reply]
Thanks a ton!
